### PR TITLE
Fixes #1363

### LINKED ISSUE #1363: Coordinated Discharge Streams GIS layer
See if we can update IN coordained reach map layer to new layer, [https://services.arcgis.com/2Mcx1M1MORKfBNPM/arcgis/rest/services/Indiana_Coordinated_Discharge_view/FeatureServer](https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fservices.arcgis.com%2F2Mcx1M1MORKfBNPM%2Farcgis%2Frest%2Fservices%2FIndiana_Coordinated_Discharge_view%2FFeatureServer&data=05%7C01%7Chwavra%40usgs.gov%7C60f5174bf5c246f7280d08da6356b722%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637931521329388958%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=kZZdIdAyDuinLy4JsOlM%2FFh4ev9KTSpHQoo4mn7TA%2F0%3D&reserved=0). 

Should ask @USGSPMM3 for more information/timelines at our next check-in.

### PR DESCRIPTION
**Pull request by harper-wavra**
Uses new layer, [https://services.arcgis.com/2Mcx1M1MORKfBNPM/arcgis/rest/services/Indiana_Coordinated_Discharge_view/FeatureServer](https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fservices.arcgis.com%2F2Mcx1M1MORKfBNPM%2Farcgis%2Frest%2Fservices%2FIndiana_Coordinated_Discharge_view%2FFeatureServer&data=05%7C01%7Chwavra%40usgs.gov%7C60f5174bf5c246f7280d08da6356b722%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637931521329388958%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=kZZdIdAyDuinLy4JsOlM%2FFh4ev9KTSpHQoo4mn7TA%2F0%3D&reserved=0) for coordinated reach calculations.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,8 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased](https://github.com/USGS-WiM/StreamStats/tree/dev)
   
   ### Added
-    - Added Channel-width Methods Weighting results to downloaded CSV
+  - Added Channel-width Methods Weighting results to downloaded CSV
   ### Changed
+  - Changed Coordinated Reach to new layer, [https://services.arcgis.com/2Mcx1M1MORKfBNPM/arcgis/rest/services/Indiana_Coordinated_Discharge_view/FeatureServer](https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fservices.arcgis.com%2F2Mcx1M1MORKfBNPM%2Farcgis%2Frest%2Fservices%2FIndiana_Coordinated_Discharge_view%2FFeatureServer&data=05%7C01%7Chwavra%40usgs.gov%7C60f5174bf5c246f7280d08da6356b722%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637931521329388958%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=kZZdIdAyDuinLy4JsOlM%2FFh4ev9KTSpHQoo4mn7TA%2F0%3D&reserved=0).
```
Comment (amedenblik) [inline CHANGELOG.md:12]: Change the hyperlink, since it's currently an Outlook protection link

File: src/Services/StudyAreaService.ts [hunk 2]
```diff
@@ -748,9 +748,9 @@ module StreamStats.Services {
                     boundingBox[index] = destination.geometry.coordinates[index % 2 == 0 ? 0 : 1];
                 });
 
-                var outFields = "eqWithStrID.Stream_Name,eqWithStrID.StreamID_ID,eqWithStrID.BASIN_NAME,eqWithStrID.DVA_EQ_ID,eqWithStrID.a10,eqWithStrID.b10,eqWithStrID.a25,eqWithStrID.b25,eqWithStrID.a50,eqWithStrID.b50,eqWithStrID.a100,eqWithStrID.b100,eqWithStrID.a500,eqWithStrID.b500";
-                var url = configuration.baseurls['StreamStatsMapServices'] + configuration.queryparams['coordinatedReachQueryService']
-                    .format(this.selectedStudyArea.RegionID.toLowerCase(), boundingBox[0], boundingBox[1], boundingBox[2], boundingBox[3], this.selectedStudyArea.Pourpoint.crs, outFields);
+                var outFields = "Stream_Name,StreamID_ID,BASIN_NAME,DVA_EQ_ID,a10,b10,a25,b25,a50,b50,a100,b100,a500,b500";
+                var url = 'https://services.arcgis.com/2Mcx1M1MORKfBNPM/' + configuration.queryparams['coordinatedReachQueryService']
```
Comment (amedenblik) [inline src/Services/StudyAreaService.ts:752]: I would change the "coordinatedReachQueryService" link in the appConfig to be "https://services.arcgis.com/2Mcx1M1MORKfBNPM/ArcGIS/rest/services/Indiana_Coordinated_Discharge_view/FeatureServer/2/query?geometry={0},{1},{2},{3}&geometryType=esriGeometryEnvelope&inSR={4}&spatialRel=esriSpatialRelIntersects&outFields={5}&returnGeometry=false&returnIdsOnly=false&returnCountOnly=false&returnZ=false&returnM=false&f=pjson" and then change this line to be: `var url = + configuration.queryparams['coordinatedReachQueryService']`

File: src/appConfig.js [hunk 3]
```diff
@@ -56,7 +56,7 @@ configuration.queryparams =
         'Wateruse': '/summary?year={0}&endyear={1}&includePermits={2}&computeReturns={3}&computeDomestic={4}',
         'WateruseSourceCSV': '/summary/bysource?year={0}&endyear={1}&includePermits={2}&computeReturns={3}&computeDomestic={4}',
         'WateruseConfig': '/regions/{0}/config',
-        'coordinatedReachQueryService': '/arcgis/rest/services/coordinatedreaches/{0}/MapServer/0/query?geometry={1},{2},{3},{4}&geometryType=esriGeometryEnvelope&inSR={5}&spatialRel=esriSpatialRelIntersects&outFields={6}&returnGeometry=false&returnIdsOnly=false&returnCountOnly=false&returnZ=false&returnM=false&returnDistinctValues=false&f=pjson',
+        'coordinatedReachQueryService': 'ArcGIS/rest/services/Indiana_Coordinated_Discharge_view/FeatureServer/2/query?geometry={0},{1},{2},{3}&geometryType=esriGeometryEnvelope&inSR={4}&spatialRel=esriSpatialRelIntersects&outFields={5}&returnGeometry=false&returnIdsOnly=false&returnCountOnly=false&returnZ=false&returnM=false&f=pjson',
```
Comment (amedenblik) [inline src/appConfig.js:59]: I would change this line to: `'coordinatedReachQueryService': 'https://services.arcgis.com/2Mcx1M1MORKfBNPM/ArcGIS/rest/services/Indiana_Coordinated_Discharge_view/FeatureServer/2/query?geometry={0},{1},{2},{3}&geometryType=esriGeometryEnvelope&inSR={4}&spatialRel=esriSpatialRelIntersects&outFields={5}&returnGeometry=false&returnIdsOnly=false&returnCountOnly=false&returnZ=false&returnM=false&f=pjson',`

File: dist/appConfig.js [hunk 4]
```diff
@@ -269,15 +269,24 @@ configuration.regions = [
         "RegionID": "IN", "Name": "Indiana", "Bounds": [[37.776224, -88.10149], [41.76554, -84.787446]], "Layers": {
             "IN_Reaches": {
                 "name": "Coordinated Reaches",
-                "url": configuration.baseurls['StreamStatsMapServices'] + "/arcgis/rest/services/coordinatedreaches/in/MapServer",
-                "type": 'agsDynamic',
+                "url": "https://services.arcgis.com/2Mcx1M1MORKfBNPM/ArcGIS/rest/services/Indiana_Coordinated_Discharge_view/FeatureServer/2",
+                "type": 'agsFeature',
                 "visible": true,
                 "layerOptions": {
```
Comment (amedenblik) [inline dist/appConfig.js:275]: Is it possible to change the color of the layer here? It is currently blue, making it hard to see compared to the streams. We may also be able to ask Indiana DNR to change the symbiology in their published layer. 

image

**Review by amedenblik — CHANGES_REQUESTED** [review]
Seems to work well!

File: dist/appConfig.js (hunk 4 above)
Comment (harper-wavra) [reply dist/appConfig.js:275]: I can look into it, but when we were trying to change the symbiology o the old layer we ended up just changing it in the map service.

Comment (amedenblik) [reply dist/appConfig.js:275]: Yeah let's just ask Indiana DNR if it's possible for them to change it.

**Comment by harper-wavra** [comment]
Closing until we have more information from IN.

**Pull request closed without merging**